README.md

```diff
@@ -1,7 +1,7 @@
-Config Files Gem
-================
-[![Code Climate](https://codeclimate.com/github/jreidinger/config_files/badges/gpa.svg)](https://codeclimate.com/github/jreidinger/config_files)
-[![Coverage Status](https://coveralls.io/repos/jreidinger/config_files/badge.svg?branch=master&service=github)](https://coveralls.io/github/jreidinger/config_files?branch=master)
+Config Files Api Gem
+=====================
+[![Code Climate](https://codeclimate.com/github/config-files-api/config_files_api/badges/gpa.svg)](https://codeclimate.com/github/config-files-api/config_files_api)
+[![Coverage Status](https://coveralls.io/repos/config-files-api/config_files_api/badge.svg?branch=master&service=github)](https://coveralls.io/github/config-files-api/config_files_api?branch=master)
 Idea of gem is to have user friendly component based way to access and modify
 configuration files. There is basically three layers.
 
```

Comment:
actually maybe it is not limited to linux system, so generic files in a system ?